subprojects/building-groovy-libraries/contents/index.adoc

```diff
@@ -58,7 +58,7 @@ Source package (default: demo):
 
 
 > Task :init
-Get more help with your project: https://docs.gradle.org/4.10.3/userguide/tutorial_groovy_projects.html
+Get more help with your project: https://docs.gradle.org/6.0/userguide/tutorial_groovy_projects.html
```

Comment:
Why would we want to link 6.0 instead of the most recent?

Reply:
That's actually a tested output. So it needs to match what the minimal version declared for this guide is.